### ISSUE TITLE
Can't get ImageGallery to show the images

### ISSUE THREAD

**Issue by andruxnet** [shoutem/ui#442]
TLDR;

See: https://stackoverflow.com/questions/57024425/shoutem-ui-imagegallery-component-not-loading-the-images

I created an ImageCarousel component where I use the ImageGallery and ImageGalleryOverlay components, but it only shows titles and descriptions, not the images.

`images` is an array the same as in the docs with `{ source: uri, title, description }`
`      <ImageGallery
        data={images}
        selectedIndex={imageIndex}
        renderImageOverlay={this.renderImageOverlay}
      />`

`this.renderImageOverlay` is exactly the same as in the docs on https://shoutem.github.io/docs/ui-toolkit/components/image-gallery

I already ran `react-native link`, it says all good but the problem persists.

What am I missing?
